remix-analyzer/src/solidity-analyzer/index.ts

```diff
@@ -18,8 +18,8 @@ export default class staticAnalysisRunner {
    */
   run (compilationResult: CompilationResult, toRun: number[], callback: ((reports: AnalysisReport[]) => void)): void {
     const modules: ModuleObj[] = toRun.map((i) => {
-      const m: AnalyzerModule = this.modules()[i]
-      return { 'name': m.name, 'mod': m }
+      const m = this.modules()[i]
+      return { 'name': new m().name, 'mod': new m() }
```

Comment:
Is it possible to instantiate it 1 time only?

Reply:
done